package.json

```diff
@@ -195,6 +205,11 @@
         "command": "one.viewer.jsonTracer",
         "title": "json tracer",
         "category": "ONE"
+      },
+      {
+        "command": "one.viewer.metadata.show",
+        "title": "Show Metadata",
+        "category": "ONE"
       }
```

Comment:
```
      {
        "command": "one.viewer.metadata.showInDefaultExplorer",
        "title": "ONE: Show Metadata",
        "category": "ONE"
      }
```

Reply:
thank you, I think it became clearer with ONE on "Show Metadata".